regulations/templates/regulations/chrome.html

```diff
@@ -138,12 +138,15 @@ <h2 class="toc-type">Search</h2>
         <div class="drawer-content">
             <form action="{% url 'chrome_search' reg_part %}" method="GET" role="search">
                 <div class="search-box">
-                    <input type="text" name="q" value="{{ q }}" id="search-input" /><button type="submit"><i class="cf-icon cf-icon-search"></i><strong class="icon-text">Search</strong></button>
+                    <input type="text" name="q" value="{{ q }}" id="search-input" title="Search term"/>
+                    <button type="submit">
+                        <i class="cf-icon cf-icon-search"></i><strong class="icon-text">Search</strong>
```

Comment:
My thoughts were to create several PRs for the different types of infractions I"m seeing - no labels / incorrectly nesting headers / <i></i> - to make them a little more easier to review.  The input ones seem more of a no-brainer - the others will likely involve templates and css changes.